test/validation/api/align/align.c

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024 Nokia
+ */
+
+#include <odp_api.h>
+#include <odp_cunit_common.h>
+
+#include <stddef.h>
+#include <stdint.h>
+
+typedef struct ODP_PACKED {
+	uint8_t a;
+	uint8_t b;
+	uint16_t c;
+	uint32_t d;
+} test_type_t;
```

Comment:
We could also have a padded structure to test that `ODP_PACKED` would remove the alignment padding.

Reply:
Second test structure added in V2.